build/ferretdb/evaluation/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+test -n "$POSTGRES_USER" || (echo "POSTGRES_USER must be set" && false)
+test -n "$POSTGRES_PASSWORD" || (echo "POSTGRES_PASSWORD must be set" && false)
+
```

Comment:
We should test more. For example, `POSTGRES_DB` must be `postgres` or unset, in which case `POSTGRES_USER` must be `postgres` or unset

Reply:
Thanks, updated :hugs: